resources/views/loadouts/index.blade.php

```diff
@@ -1,51 +1,30 @@
 @extends('layouts.app')
 
+@section('styles')
+<link
+  rel="stylesheet"
+  href="https://cdn.jsdelivr.net/npm/choices.js/public/assets/styles/choices.min.css"
```

Comment:
Seems like we have `npm install`'ed these (and others) but then we are using a CDN. I vote we remove all the additional package json dependencies and leave this for now. Thoughts?

Reply:
I took the `package.json` out back and gave it what for